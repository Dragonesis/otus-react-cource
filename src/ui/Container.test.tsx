import React from 'react'
import { render } from '@testing-library/react'

import { Container } from './Container'

describe('### Container', () => {
  test('Container renders correctly', () => {
    const { asFragment } = render(<Container />)
    expect(asFragment()).toMatchSnapshot()
  })
})
