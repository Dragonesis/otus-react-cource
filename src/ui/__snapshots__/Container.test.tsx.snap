// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`### Container Container renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  max-width: 1000px;
  width: 100%;
  margin-left: auto;
  margin-right: auto;
}

<div
    class="emotion-0"
  />
</DocumentFragment>
`;
